app/views/hub/notes/_note.html.erb

```diff
@@ -1,13 +1,14 @@
-<li class="note message message--<%= note.user == current_user ? "outgoing" : "incoming"%>" id="<%= "last-item" if local_assigns[:is_last_item] %>">
-  <div class="message__heading">
-    <span class="message__author">
-      <%= note.user.name %>
-    </span>
-    <span class="message__time">
-      <%= long_formatted_datetime(note.created_at) %>
-    </span>
+<div class="note <%= "system_note" if note.is_a? SystemNote %>" id="<%= "last-item" if local_assigns[:is_last_item] %>">
+  <div class="note__heading">
+    <div class="note__author">
+      <%= note&.user&.name %>
+    </div>
+    <div class="note__time">
+      <%= timestamp(note.created_at) %>
+    </div>
   </div>
-  <div class="message__body">
-    <%= note.body %>
+
+  <div class="note__body">
+    <%= simple_format(note.body) %>
```

Comment:
really happy to see this usage of `simple_format`!! Awesome! Should we be using it elsewhere?